src/Microsoft.Framework.Localization/LocalizationOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Localization
+{
+    /// <summary>
+    /// Provides programmatic configuration for localization.
+    /// </summary>
+    public class LocalizationOptions
+    {
+        /// <summary>
+        /// The relative path under application root where resource files are located.
+        /// </summary>
+        public string ResourcesPath { get; set; }
```

Comment:
No since `ResourceManager` that uses this is a private field but we will be using this feature in MVC repo.